webapp/pages/settings/my-social-media.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <ds-card :header="$t('settings.social-media.name')">
+    <ds-space
+      v-if="socialMediaLinks"
+      margin-top="base"
+      margin="x-small"
+    >
+      <ds-list>
+        <ds-list-item
+          v-for="link in socialMediaLinks"
+          :key="link.url"
+        >
+          <a :href="link.url">
+            <img
+              :src="link.favicon"
+              alt=""
+            >
+            {{ link.url }}
+          </a>
+        </ds-list-item>
+      </ds-list>
+    </ds-space>
+    <div>
+      <ds-input
+        v-model="value"
+        placeholder="Add social media url"
+        name="social-media"
+        :schema="{type: 'url'}"
+      />
+    </div>
+    <ds-space margin-top="base">
+      <div>
+        <ds-button
+          primary
+          @click="handleAddSocialMedia"
+        >
+          {{ $t('settings.social-media.submit') }}
+        </ds-button>
+      </div>
+    </ds-space>
+  </ds-card>
+</template>
+<script>
+import gql from 'graphql-tag'
+import { mapGetters } from 'vuex'
+
+export default {
+  data() {
+    return {
+      value: ''
+    }
+  },
+  computed: {
+    ...mapGetters({
+      currentUser: 'auth/user'
+    }),
```

Comment:
@mattwr18 you asked why the page does not get updated: You have to update `currentUser`. Import `mapMutations` put it in `methods` and use it when `apollo` successfully executes the mutation.

Reply:
we tried doing that, but I think we didn't push up the code to have a look if we implemented it correctly... I can add it back and push it up

Reply:
@aonomike and I were able to get it updating the store and now the icons and links are showing up when the user adds the social media url!! 

the favicons are quite small from twitter and facebook for example... not sure if there is a way to size them without blurring them... for the time being, I have added a 16x16 constraint on all icons, but it seems to small to me... also, we obviously will work on the other review items you brought up @roschaefer like limiting how many a user can add, validating the url in the back end, etc... 
![Screenshot at 2019-03-28 15:46:49](https://user-images.githubusercontent.com/26943915/55184590-4c041100-5171-11e9-8fa5-2d504ca278d7.png)

Reply:
It's OK, I think we can resize them later. For styles and visuals you can also consult @appinteractive. 

It does not look so bad to me honestly.